arch/x86/kvm/pkvm/entry.S

```diff
@@ -83,3 +84,16 @@ SYM_CODE_START_NOALIGN(x86_verw_sel)
 SYM_CODE_END(x86_verw_sel);
 
 .popsection
+
+/* Clobbers AX, CX, DX */
+SYM_FUNC_START(write_ibpb)
+	ANNOTATE_NOENDBR
+	movl	$MSR_IA32_PRED_CMD, %ecx
+	movl	_ASM_RIP(x86_pred_cmd), %eax
+	xorl	%edx, %edx
+	wrmsr
+
+	/* Make sure IBPB clears return stack preductions too. */
+	FILL_RETURN_BUFFER %rax, RSB_CLEAR_LOOPS, X86_BUG_IBPB_NO_RET
```

Comment:
Just a fun note: when I'm running a Linux VM in qemu, for some reason the VM thinks that the CPU has `ibpb_no_ret` bug (so in particular, when I'm running pKVM in qemu, it says: `pkvm: unmitigated cpu bug ibpb_no_ret`).

And looks like we are actually effectively mitigating X86_BUG_IBPB_NO_RET here, so we could even add it to the list of CPU bugs supported by pKVM. (But I'm not saying we should do that. Looks like this bug is AMD-specific anyway, so we don't need to care.)